README.md

```diff
@@ -16,45 +28,75 @@ video:
+version. Windows local profiling is currently only supported partially and major
+features, such as dynamic instrumentation, are not yet implemented. It is
+possible however to profile Linux executable from a Windows UI instance. For
+Windows local profiling, please use the released
+[binaries](https://github.com/google/orbit/releases).
 
 ## Build
 
-Please have a look at the first three sections of our [development documentation](DEVELOPMENT.md). It describes how to build Orbit and what Compilers, Platforms, and Tools are supported and needed.
+Please have a look at the first three sections of our
+[development documentation](DEVELOPMENT.md). It describes how to build Orbit and
+what Compilers, Platforms, and Tools are supported and needed.
 
 ## Workflow
 
-> **Note** And extensive documentation of the usage of Orbit can be found in our [usage documentation](documentation/DOCUMENTATION.md).
+> **Note** An extensive documentation of the usage of Orbit can be found in our
+> [usage documentation](documentation/DOCUMENTATION.md).
 
 The following describes the basic workflow of Orbit:
-1. Select a process in the list of currently running processes in the connection setup dialog, and click <b><samp>Start Session</samp></b>.
-2. The list of loaded modules will appear on the top of the <b><samp>Symbols</samp></b> tab.  If debug symbols were found for a module, it will be highlighted in blue.
-3. <b><samp>Right click</samp></b> on the module(s) for which you want to load debug information and select <b><samp>Load Symbols</samp></b>. The <b><samp>Functions</samp></b> tab will get populated.
-4. Select functions you wish to profile in the <b><samp>Functions</samp></b> tab by <b><samp>right clicking</samp></b> and choosing <b><samp>Hook</samp></b>.
-5. In the <b><samp>Capture</samp></b> tab, start profiling by pressing <kbd>F5</kbd>.  To stop profiling, press <kbd>F5</kbd> again. You can either zoom time using <kbd>W</kbd> and <kbd>S</kbd> or <kbd>ctrl</kbd> + the scroll wheel.  You can also <kbd>ctrl</kbd>+<b><samp>right-click</samp></b> and drag to zoom to a specific time range. To scale the UI, press <kbd>ctrl</kbd> + <kbd>+</kbd>/<kbd>-</kbd>. Press <kbd>SPACE</kbd> to see the last 2 seconds of capture.
-6. You can select sections of the per-thread sampling event track to get a sampling report of your selection.
+1. Select a process in the list of currently running processes in the connection
+   setup dialog, and click <b><samp>Start Session</samp></b>.
+2. The list of loaded modules will appear on the top of the
+   <b><samp>Symbols</samp></b> tab. If debug symbols were found for a module,
+   it will be highlighted in green.
+3. <b><samp>Right click</samp></b> on the module(s) for which you want to load
+   debug information and select <b><samp>Load Symbols</samp></b>. The
+   <b><samp>Functions</samp></b> tab will get populated.
+4. Select functions you wish to profile in the <b><samp>Functions</samp></b> tab
```

Comment:
"profile"

We should specify this is dynamic instrumentation.

Reply:
Done.